jsapp/js/hooks/useApiFetcher.hook.ts

```diff
@@ -102,6 +105,9 @@ export const useApiFetcher = <Type>(
   return [response, loadFetcher, {...status, setIsInitialLoad}];
 };
 
+/**
```

Comment:
`withApiFetcher` is used by class components, so maybe we should add more instructions on whether the component should be converted to functional. I don't think any new implementation would use this, tho.